### PR TITLE
Add migros-bank-e-banking.app v8.1.2500

### PR DESCRIPTION
**Pull request by GETandSELECT**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/migros-bank-e-banking.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask "migros-bank-e-banking" do
+  version "8.1.2500"
+  sha256 "fea31fa340912cd5cab6332fd1b09c3ad366eade655590e7be3c0b42cfce04ad"
+
+  url "https://download.migrosbank.ch/mid/MigrosBank-EBanking-Mac-8.1.2500.pkg"
```
Comment (miccal) [inline Casks/migros-bank-e-banking.rb:5]: Please change this to:
```
url "https://download.migrosbank.ch/mid/MigrosBank-EBanking-Mac-#{version}.pkg"
```

File: Casks/migros-bank-e-banking.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  name "Migros Bank E-Banking"
+  desc "Client for Migros Bank E-Banking (Switzerland)"
+  homepage "https://www.migrosbank.ch/de/privatpersonen/e-banking/zugriff.html"
+
+  pkg "MigrosBank-EBanking-Mac-8.1.2500.pkg"
+
+  uninstall pkgutil: [
+    "com.kobil.midappqt.PROD",
+  ]
```
Comment (miccal) [inline Casks/migros-bank-e-banking.rb:14]: Please change this to:
```
uninstall pkgutil: "com.kobil.midappqt.PROD"
```

**Comment by GETandSELECT** [comment]
thank you for improvements.

All checks passed :-)

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> + The Cask is too obscure. Examples:
>   + A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 30 forks, 30 watchers, and 75 stars).
>   + [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021).

This is overly specific, even more so than eID software.

**Pull request closed without merging**